win32ss/user/user32/user32.spec

```
@@ -1,752 +1,752 @@
+@ stdcall SetWindowPlacement(long ptr) NtUserSetWindowPlacement
+@ stdcall SetWindowPos(long long long long long long long) NtUserSetWindowPos
+@ stdcall SetWindowRgn(long long long)
+@ stdcall SetWindowStationUser(long long long long)
+@ stdcall SetWindowTextA(long str)
+@ stdcall SetWindowTextW(long wstr)
+@ stdcall SetWindowWord(long long long) ; Direct call NtUserSetWindowWord
+@ stdcall SetWindowsHookA(long ptr)
+@ stdcall SetWindowsHookExA(long long long long)
+@ stdcall SetWindowsHookExW(long long long long)
+@ stdcall SetWindowsHookW(long ptr)
+@ stdcall ShowCaret(long) NtUserShowCaret
+@ stdcall ShowCursor(long)
+@ stdcall ShowOwnedPopups(long long)
+@ stdcall ShowScrollBar(long long long) NtUserShowScrollBar
+@ stdcall ShowStartGlass(long)
+@ stdcall ShowWindow(long long) NtUserShowWindow
+@ stdcall ShowWindowAsync(long long) NtUserShowWindowAsync
+@ stdcall SoftModalMessageBox(ptr)
+@ stdcall SubtractRect(ptr ptr ptr)
+@ stdcall SwapMouseButton(long)
+@ stdcall SwitchDesktop(long) NtUserSwitchDesktop
+@ stdcall SwitchToThisWindow(long long)
+@ stdcall SystemParametersInfoA(long long ptr long)
+@ stdcall SystemParametersInfoW(long long ptr long)
+@ stdcall TabbedTextOutA(long long long str long long ptr long)
+@ stdcall TabbedTextOutW(long long long wstr long long ptr long)
+@ stdcall TileChildWindows(long long)
+@ stdcall TileWindows(long long ptr long ptr)
+@ stdcall ToAscii(long long ptr ptr long)
+@ stdcall ToAsciiEx(long long ptr ptr long long)
+@ stdcall ToUnicode(long long ptr ptr long long)
+@ stdcall ToUnicodeEx(long long ptr ptr long long long)
+@ stdcall TrackMouseEvent(ptr) NtUserTrackMouseEvent
+@ stdcall TrackPopupMenu(long long long long long long ptr)
+@ stdcall TrackPopupMenuEx(long long long long long ptr) NtUserTrackPopupMenuEx
+@ stdcall TranslateAccelerator(long long ptr) TranslateAcceleratorA
+@ stdcall TranslateAcceleratorA(long long ptr)
+@ stdcall TranslateAcceleratorW(long long ptr)
+@ stdcall TranslateMDISysAccel(long ptr)
+@ stdcall TranslateMessage(ptr)
+@ stdcall TranslateMessageEx(ptr long)
+@ stdcall UnhookWinEvent(long) NtUserUnhookWinEvent
+@ stdcall UnhookWindowsHook(long ptr)
+@ stdcall UnhookWindowsHookEx(long) NtUserUnhookWindowsHookEx
+@ stdcall UnionRect(ptr ptr ptr)
+@ stdcall UnloadKeyboardLayout(long)
+@ stdcall UnlockWindowStation(long) NtUserUnlockWindowStation
+@ stdcall UnpackDDElParam(long long ptr ptr)
+@ stdcall UnregisterClassA(str long)
+@ stdcall UnregisterClassW(wstr long)
+@ stdcall UnregisterDeviceNotification(long)
+@ stdcall UnregisterHotKey(long long) NtUserUnregisterHotKey
+@ stdcall UnregisterMessagePumpHook()
+@ stdcall UnregisterUserApiHook() NtUserUnregisterUserApiHook
+@ stdcall UpdateLayeredWindow(long long ptr ptr long ptr long ptr long)
+@ stdcall UpdateLayeredWindowIndirect(long ptr)
+@ stdcall UpdatePerUserSystemParameters(long long)
+@ stdcall UpdateWindow(long)
+@ stdcall User32InitializeImmEntryTable(ptr)
+@ stdcall UserClientDllInitialize(long long ptr) DllMain
```

The first parameter of UserClientDllInitialize i.e. DllMain is a HANDLE / HINSTANCE, so it should stay `ptr`.
```suggestion
@ stdcall UserClientDllInitialize(ptr long ptr) DllMain
```

---

fixed